### PR TITLE
Calculate Rotation angle on ALFView

### LINKED ISSUE #34569: ALFView - Calculate `R` and display clearly
**Description**
The value for `R` should be calculated from `2θ` and `V`, and this should be displayed clearly on the ALFView interface.

```
R = V / (2sinθ)
```

At the moment, users have to calculate this value manually. This is true for ALFplus too.

**Solutions Considered**
TBD

### PR DESCRIPTION
**Pull request by robertapplin**
**Description of work.**
This PR adds the calculation of the Rotation angle to the ALFView interface. If a Fit hasn't been performed, then the Rotation angle will be blank and a red asterisk displayed next to it.

**To test:**
Select ALF instrument in settings, and turn on data archive.
Open ALFView
Load run 82301
Click on Pick tab
Expand Rebin section. Rebin with 5.5,0.01,6
Click the Rectangle region selection tool at the top of the Pick tab
Select the two tubes in the centre with the yellow peak. The RHS plot should be updated.
Click Fit, and a Successful fit status label should appear.
You should also see a value for the "Rotation" angle appears.
If you Reset the plot with the button in the top right then the Rotation angle should go blank, and a red asterisk appear next to the box.

Fixes #34569

*This does not require release notes* because **release notes will be added later**

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: qt/scientific_interfaces/Direct/ALFAnalysisView.cpp [hunk 1]
```diff
@@ -190,6 +192,22 @@ void ALFAnalysisView::setupPeakCentreWidget(QGridLayout *layout, double const ce
...
+  layout->addWidget(new QLabel(""), 4, 4);
+}
+
+void ALFAnalysisView::setupRotationAngleWidget(QGridLayout *layout) {
+  m_rotationAngle = new QLineEdit("-");
+  m_rotationAngle->setReadOnly(true);
+  m_rotationAngle->setToolTip(ROTATION_ANGLE_TOOLTIP);
+  m_fitRequired = new QLabel("*");
+  m_fitRequired->setToolTip("A Fit to find the peak centre is required.");
+  m_fitRequired->setStyleSheet("QLabel { color: red; }");
```
Comment (jhaigh0) [inline qt/scientific_interfaces/Direct/ALFAnalysisView.cpp:206]: The asterisk appears quite small when testing. Could it be a little bigger? If this is consistent with the rest of the interface, then don't worry.
image

**Review by jhaigh0 — CHANGES_REQUESTED** [review]
Good change with neat code. Only a minor thing about the asterisk size. Tests look sensible with good coverage of possible interactions.

File: qt/scientific_interfaces/Direct/ALFAnalysisView.cpp (hunk 1 above)
Comment (robertapplin) [reply qt/scientific_interfaces/Direct/ALFAnalysisView.cpp:206]: Thanks for reviewing! I think for this particular case I want the User not to notice the asterisk unless they are looking directly at the "Rotation" box and wondering why there isn't a value - so a small asterisk might be preferred

Also I think if we change the font-size of this individual asterisk then it won't respond to the User changing the font-size from the settings GUI. In general, I think the size is consistent with other interfaces as red asterisks are sized based on the font-size

Comment (jhaigh0) [reply qt/scientific_interfaces/Direct/ALFAnalysisView.cpp:206]: Makes sense, thanks for the additional context 👍

**Review by jhaigh0 — APPROVED** [review]

**Pull request merged**